README.md

```diff
@@ -15,9 +15,17 @@ CloudAI operates on four main schemas:
 These schemas enable CloudAI to be flexible and compatible with different systems and configurations.
 
 ## Quick Start
-First, ensure you have access to the docker repository. This is a manual step:
-- In https://ngc.nvidia.com/signin go to setup -> generate api key
-- Set up enroot credentials properly. Ensure you have the correct credentials under `~/.config/enroot/.credentials`:
+First, ensure you have access to the Docker repository. Follow these steps:
```

Comment:
Can we move this guidance into a separate section, `## Setup access to private NGC registry` to highlight where it is applicable?

Reply:
Then, we need to provide a section title for the following steps. You can take my PR and update it or create a new PR.

Reply:
Everything else can remain under "Quick start"

Reply:
Updated.